app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java

```diff
@@ -127,6 +127,9 @@ public static void userChangedQuality(int selectedQuality) {
         userChangedQuality = true;
     }
 
+    public static void newVideoStarted(String useless) {
+        ReVancedUtils.setNewVideo(true);
```

Comment:
Why is `ReVancedUtils` being used here? Shouldn't you set a field inside this class as you only need the video id here?

Reply:
I only need to know if a new video is started or not. video id is useless in this patch.

Reply:
Is there any use to call  `ReVancedUtils.setNewVideo(true);`, create a boolean field and work with that one.

Reply:
done